action.yml

```diff
@@ -24,6 +24,10 @@ inputs:
         - base: backport PR's base branch
         - number: original PR's number
     default: "backport-<%= number %>-to-<%= base %>"
+  team_reviews:
```

Comment:
Just leaving this open so that the relevant teams can specify a required team reviewer if desired?

Reply:
Oh nvm answered in PR description

Reply:
Yeah parameterizing it so it can be overriden, but the default stays the same as it currently works